specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
The model we've mostly discussed for the general case in CORLA2 would still have just one table ala `county_contest_table`, though perhaps it could be called `collection_contest_table` to be more general, following Ron Rivest's terminology for the smallest jurisdiction that maintains a actual physical collection of ballots and can inspect a given sample of them. Contests can involve a subset of that collection, or involve several collections, all the way to the United Federation of Planets spread across all the galaxies, to the extent that we could communicate fast enough...